models/curvefi/celo/curvefi_celo_pools.sql

```diff
@@ -0,0 +1,22 @@
+{{
+    config(
+        tags = ['dunesql'],
+        alias = alias('pools'),
+        materialized = 'table',
+        unique_key = ['version', 'tokenid', 'token', 'pool'],
+        post_hook = '{{ expose_spells(\'["celo"]\',
```

Comment:
another thing we're cleaning up a bit:
does this spell need to be exposed on data explorer?

rule of thumb on this:
is the intention to query directly on the app often (then expose), or is it just a stepping stone to be used in downstream spells (then don't expose)?

Reply:
yep, this spell actually doesn't have to be exposed - will remove